src/shared/codelens/typescriptCodeLensProvider.ts

```diff
@@ -59,8 +56,8 @@ export function initialize({
     configuration,
     outputChannel: toolkitOutputChannel,
     processInvoker = new DefaultSamCliProcessInvoker(),
-    taskInvoker = new DefaultSamCliTaskInvoker(),
-    telemetryService
+    localInvokeCommand = new DefaultSamLocalInvokeCommand(getChannelLogger(toolkitOutputChannel)),
```

Comment:
Out of scope for this change, but non-exported functions shouldn't have default values for 'context'-like parameters. Unit tests won't call this function directly, and omitting the default value forces the compiler to yell at us if we forget to pass in a value from the caller.